Fabric.Identity.API/scripts/Register.ps1

```diff
@@ -435,6 +435,7 @@ function Invoke-WriteSecretToConfig($service, $secret, $configPath){
     
     if(!([string]::IsNullOrWhiteSpace($service.secretConfig))){
         Add-WebConfigAppSetting -webConfigLocation $configPath -settingKey $service.secretConfig -settingValue $secret | Out-Null
+		Write-Success "    Wrote secret to $service.serviceName configuration file as key=$service.secretConfig, value=$secret."
```

Comment:
Should we add the path of the file to the log message also, we have it in $configPath I think.

Reply:
Yes - I meant to put that in. Thanks.